src/mam4xx/mo_sethet.hpp

```diff
@@ -0,0 +1,179 @@
+#ifndef MAM4XX_MO_SETHET_HPP
+#define MAM4XX_MO_SETHET_HPP
+
+#include <haero/math.hpp>
+#include <mam4xx/gas_chem.hpp>
+#include <mam4xx/mam4_types.hpp>
+#include <mam4xx/mo_chm_diags.hpp>
+#include <mam4xx/utils.hpp>
+
+namespace mam4 {
+
+namespace mo_sethet {
+
+// const int ktop = ConvProc::Config::ktop;
+constexpr Real avo = haero::Constants::avogadro;
+const Real pi = haero::Constants::pi;
+constexpr Real rgrav =
+    mo_chm_diags::rgrav; // reciprocal of acceleration of gravity ~ m/s^2
+constexpr const int gas_pcnst = gas_chemistry::gas_pcnst;
+const Real boltz_cgs = haero::Constants::boltzmann * 1.e7; // erg/K
+// number of vertical levels
+constexpr const int pver = mam4::nlev;
+
+// FIXME: BAD CONSTANT
+// mass of water vapor [amu] //convert to g/mol from kg/mol
+constexpr const Real mass_h2o = haero::Constants::molec_weight_h2o * 1000;
+constexpr const Real cm3_2_m3 = 1.0e-6; // convert cm^3 to m^3
+constexpr const Real liter_per_gram = 1.0e-3;
+constexpr const Real avo2 =
+    avo * liter_per_gram * cm3_2_m3; // [liter/gm/mol*(m/cm)^3]
```

Comment:
I've never seen this before, but is doubling up `constexpr const` a usual thing?

Reply:
I'll be honest, I didn't notice that I did that - I think `constexpr` is sufficient but apparently the extra `const` didn't make anything complain

Reply:
That is a lotta `const`! Maybe it's good to remove the extra `const` just to prevent future `const`-fusion